frontend/projects/valtimo/plugin-management/src/lib/components/plugin-external-edit-modal/plugin-external-edit-modal.component.scss

```
@@ -30,3 +30,14 @@
   margin-top: 16px;
   margin-bottom: 8px;
 }
+
+.plugin-external-edit-modal__awaiting-host {
+  display: block;
+  margin-bottom: var(--cds-spacing-05);
+}
+
+:host ::ng-deep .plugin-external-edit-modal__awaiting-host {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)